providers/gce/gce_loadbalancer_internal.go

```diff
@@ -254,6 +256,54 @@ func (g *Cloud) ensureInternalLoadBalancer(clusterName, clusterID string, svc *v
 	return status, nil
 }
 
+func removeNodesInNonDefaultNetworks(nodes []*v1.Node, defaultSubnetName string) []*v1.Node {
+	var newList []*v1.Node
+	var skippedNodes []string
+	for _, node := range nodes {
+		subnetLabel, ok := node.Labels[labelGKESubnetworkName]
```

Comment:
factor out the default subnet as a predicate

isDefaultSubnet()

and document it appropriately.